packages/pilot/src/containers/TransactionDetails/EventDetails/style.css

```
@@ -9,11 +9,19 @@
     text-transform: capitalize;
   }
 
-  & .eventDescription {
+  & .eventDetails {
     line-height: 1.6;
 
     & p {
       margin: 0;
     }
   }
+
+  & .eventDescription {
+    line-height: 1;
+
+    & p {
+      margin: 0 0 var(--spacing-small) 0;
```

por que não apenas isso?
```suggestion
      margin-bottom: var(--spacing-small);
```

---

@miglsoares fiz com `margin` porque imaginei que deixaria mais seguro e claro que somente a margem de baixo deveria existir, acha desnecessário?